### ISSUE TITLE
Traceback: Cannot create or construct a class debug_toolbar_line_profiler.panel.DjangoDebugToolbarStats object from cProfile.Profile object

### ISSUE THREAD

**Issue by andresriancho** [dmclain/django-debug-toolbar-line-profiler#5]
The environment:
```
Request Method: GET
Request URL: http://127.0.0.1:8000/docs/api/search/?q=domains

Django Version: 1.5.1
Python Version: 2.7.3
Installed Applications:
('django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.sites',
 'django.contrib.messages',
 'django.contrib.staticfiles',
 'registration',
 'passwords',
 'axes',
 'tastypie',
 'haystack',
 'custom_user',
 'djangosecure',
 'captcha',
 'django_crypto',
 'djcelery',
 'django_gravatar',
 'favicon',
 'south',
 'constance',
 'constance.backends.database',
 'storages',
 'docs',
 'sphinxdoc',
 'django_nose',
 'user',
 'api_key',
 'profile',
 'contact',
 'account',
 'bootstrap_toolkit',
 'django_extensions',
 'suit',
 'suit_redactor',
 'django.contrib.admin',
 'django.contrib.humanize',
 'django.contrib.webdesign',
 'django.contrib.sitemaps',
 'debug_toolbar',
 'debug_toolbar_line_profiler',
 'template_timings_panel')
Installed Middleware:
('api.middleware.strip_cookies.StripCookiesMiddleware',
 'djangosecure.middleware.SecurityMiddleware',
 'api.middleware.defaultsite.DefaultSiteMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'user.auth.middleware.ActiveAndNoErrorUserMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware',
 'axes.middleware.FailedLoginMiddleware',
 'pagination_bootstrap.middleware.PaginationMiddleware',
 'debug_toolbar.middleware.DebugToolbarMiddleware',
 'api.middleware.sqlquery_log.QueryCountDebugMiddleware')
```

DDT toolbars versions:
```
django-debug-toolbar==1.2.1
django-debug-toolbar-line-profiler==0.4.0
django-debug-toolbar-template-timings==0.6.4
```

And the traceback:
```pytb
Traceback:
File "/home/pablo/PycharmProjects/virtual-envs/scan-api/local/lib/python2.7/site-packages/django/core/handlers/base.py" in get_response
  187.                 response = middleware_method(request, response)
File "/home/pablo/PycharmProjects/virtual-envs/scan-api/local/lib/python2.7/site-packages/debug_toolbar/middleware.py" in process_response
  86.             new_response = panel.process_response(request, response)
File "/home/pablo/PycharmProjects/virtual-envs/scan-api/local/lib/python2.7/site-packages/debug_toolbar_line_profiler/panel.py" in process_response
  196.         self.stats = DjangoDebugToolbarStats(self.profiler)
File "/usr/lib/python2.7/pstats.py" in __init__
  81.         self.init(arg)
File "/usr/lib/python2.7/pstats.py" in init
  95.         self.load_stats(arg)
File "/usr/lib/python2.7/pstats.py" in load_stats
  124.                               self.__class__, arg)

Exception Type: TypeError at /docs/api/search/
Exception Value: Cannot create or construct a <class debug_toolbar_line_profiler.panel.DjangoDebugToolbarStats at 0x7f49c06fadb8> 
object from '<cProfile.Profile object at 0x7f49b47c5a60>''
```

This only happens on this specific page, all other pages work well and the toolbar is shown.

**Comment by abriemme** [comment]
I have the same issue

**Comment by bwarren2** [comment]
I get this on my RSS page with the Feed syndication framework.

**Comment by mikob** [comment]
Does anyone have any clues on how this might be fixed?

**Comment by PetrDlouhy** [comment]
I got this error on Python 3. I solved this by using code on PR #7 with one addition, that I posted there in comment.

**Comment by dmclain** [comment]
PR #7 was merged, so hopefully this problem isn't occurring with the latest release.

Please try the latest version (`pip install django-debug-toolbar-line-profiler==0.5.1` at the time of this comment) and reopen if the problem is still happening. I appreciate the detail on the initial bug report and apologize for letting it linger for so long.
